### PR TITLE
Tests to help improve coverage

### PR DESCRIPTION
**Pull request by hsm207**

### CODE REVIEW

**Comment by oscardssmith** [comment]
Any chance of rebase? These look good. Sorry for letting this go ignored.

**Comment by hsm207** [comment]
> Any chance of rebase? These look good. Sorry for letting this go ignored.

Do you mean for me to do a rebase to resolve the merge conflicts? If yes, could you please direct me to some docs that show the git conventions this project follows?

**Comment by oscardssmith** [comment]
Yeah. I'm not aware of specific git conventions we follow. What do you mean by that?

**Comment by simeonschaub** [comment]
We almost always just squash merge PRs, so it doesn't really matter whether you rebase or merge upstream changes into your branch.

File: test/download.jl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+
```
Comment (vtjnash) [inline test/download.jl:3]:
```suggestion
```

**Pull request merged**